Spanish (la) translation

Firigion opened the pull request.

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "no_redo": "r Sin acciones que rehacer para el jugador %s",
+  "help_cmd_expand": "l Expande selección [magn] respecto a [pos]",
+  "brush_info_no_flags": "w sin banderas",
+  "help_make_selection": "c Usar la barita para seleccionar posiciones inicial y final",
```
```suggestion
  "help_make_selection": "c Usar la varita para seleccionar posiciones inicial y final",
```

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "brush_empty_list": "gi Sin brochas registradas hasta ahora",
+  "action_cone": "cono",
+  "action_cuboid": "cuboide",
+  "help_cmd_brush_list": "l Lista todas las prochas registradas y sus acciones",
```
```suggestion
  "help_cmd_brush_list": "l Lista todas las brochas registradas y sus acciones",
```

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "help_pagination_prev": "g Página anterior (%d)",
+  "help_cmd_copy": "l Copiar selección actual al portapapeles",
+  "help_cmd_brush_flood": "l Registra brocha para realizar un llenado de inundación con [block] empezando en el bloque seleccionado",
+  "more_moves_redo": "w Valor demasiado grande, rehaciando todas las acciones de %s",
```
```suggestion
  "more_moves_redo": "w Valor demasiado grande, rehaciendo todas las acciones de %s",
```

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "help_cmd_copy": "l Copiar selección actual al portapapeles",
+  "help_cmd_brush_flood": "l Registra brocha para realizar un llenado de inundación con [block] empezando en el bloque seleccionado",
+  "more_moves_redo": "w Valor demasiado grande, rehaciando todas las acciones de %s",
+  "no_undo_history": "w Sin historia de acciones para mostrar de %s",
```
`historia` or `historial`? (not sure, not only here)

Firigion replied:
I think `historial` might be a better translation, yeah. You know how you get stuck when trying to directly translate a word that has a really similar word in spanish :P

Ghoulboy78 replied:
ngl I can relate with Italian and french...

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "action_line": "línea",
+  "action_structure_paste": "pegado_estructura",
+  "brush_list_header": "bc === Las brochas actuales son ===",
+  "help_cmd_brush_cylinder": "l Registra brocha para crar un cilindro con [radius] y [height] a lo largo de [axis] de [block]",
```
```suggestion
  "help_cmd_brush_cylinder": "l Registra brocha para crear un cilindro con [radius] y [height] a lo largo de [axis] de [block]",
```

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "brush_list_header": "bc === Las brochas actuales son ===",
+  "help_cmd_brush_cylinder": "l Registra brocha para crar un cilindro con [radius] y [height] a lo largo de [axis] de [block]",
+  "help_cmd_brush_spray": "l Registra brocha para pintura en aerosol con el radio y la cantidad de puntos dados",
+  "more_moves_undo": "w Valor demasiado grande, deshaciando todas las acciones de %s",
```
```suggestion
  "more_moves_undo": "w Valor demasiado grande, deshaciendo todas las acciones de %s",
```

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "help_cmd_rotate": "l Rota [deg] en torno a [pos]",
+  "brush_info_action": "b  Acción: ",
+  "help_cmd_brush_pyramid": "l Registra brocha para crear pirámide con [radius] y [height] sobre eje [axis] en la dirección del signo",
+  "langs_completeness": "gi Nota: %s la traducción está completa en un %s%%: %s frases faltantes",
```
```suggestion
  "langs_completeness": "gi Nota: La traducción al %s solo está completa en un %s%%: %s frases faltantes",
```
First %s refers to the language name, so `La traducción al Español solo está completa...`

Firigion replied:
woops, I really missed this on XD

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "brush_info_action": "b  Acción: ",
+  "help_cmd_brush_pyramid": "l Registra brocha para crear pirámide con [radius] y [height] sobre eje [axis] en la dirección del signo",
+  "langs_completeness": "gi Nota: %s la traducción está completa en un %s%%: %s frases faltantes",
+  "no_undo": "r Sin acciones que deshacer de %s",
```
`para` maybe? == `No actions to undo for player %s`

Firigion replied:
Papa patata. In spanish I feel it's more natural that the actions are "from" some player.

Ghoulboy78 replied:
Yes, I agree with Firigion here. ALso, that 's how you would translate it into like Italian or French

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "brush_reach": "w El alcance de las brochas es %d bloques",
+  "help_cmd_undo_history": "l Muestra la historia de acciones deshechas",
+  "success_redo": "gi Se rechicieron %d exitosamente, rellenando %d bloques",
+  "no_longer_brush": "w %s ya no es más una brocha",
```
May be regional (ignore if so), but this doesn't sound very well to me (I think because of `más`)

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "entry_undo_1": "w %d: tipo: %s (dimensión: %s):",
+  "brush_reach": "w El alcance de las brochas es %d bloques",
+  "help_cmd_undo_history": "l Muestra la historia de acciones deshechas",
+  "success_redo": "gi Se rechicieron %d exitosamente, rellenando %d bloques",
```
I think this is missing _what_ was redid (`operaciones`/`acciones`/whatever).

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "brush_info_flags": "b Banderas: ",
+  "action_sphere": "esfera",
+  "help_cmd_paste": "l Pega portapapeles",
+  "help_cmd_brush_generic": "l Sostiene un ítem para covnertirlo en brocha",
```
I'd use a more personal form, not sure what you think.

Firigion replied:
As in `Sostén un ítem...`? Maybe. See discussion below?

Firigion replied:
Also, "Pega _el_ portapapeles". Note to self.

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "action_sphere": "esfera",
+  "help_cmd_paste": "l Pega portapapeles",
+  "help_cmd_brush_generic": "l Sostiene un ítem para covnertirlo en brocha",
+  "angel_block_bad_item": "r Bloque nagel no puede ser alg que ya está asignado a otra acción",
```
```suggestion
  "angel_block_bad_item": "r Bloque ángel no puede ser algo que ya está asignado a otra acción",
```
And maybe subjunctive (`esté`).

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "help_cmd_paste": "l Pega portapapeles",
+  "help_cmd_brush_generic": "l Sostiene un ítem para covnertirlo en brocha",
+  "angel_block_bad_item": "r Bloque nagel no puede ser alg que ya está asignado a otra acción",
+  "move_selection_no_player_error": "r Para mover la selección en la dirección del jugadro, hace falta un jugador",
```
```suggestion
  "move_selection_no_player_error": "r Para mover la selección en la dirección del jugador, hace falta un jugador",
```

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "action_cube": "cubo",
+  "new_wand": "wi La varita de la aplicación ahora es %s, usar con cuidado.",
+  "help_cmd_brush_spray_held": "l Registra brocha para pintura en aerosol con el bloque sostenido en la mano secundaria",
+  "help_welcome": "c Bienvenidos al menú de ayuda de Scarpet-Edit!",
```
I'd use singular. Not sure how to handle gender, though.
```suggestion
  "help_welcome": "c Bienvenido al menú de ayuda de Scarpet-Edit!",
```

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "help_cmd_expand_tooltip": "g Expande la selección en [magnitude] centrado en [pos]",
+  "help_cmd_brush_clear": "l Desregistra ítem actual como brocha",
+  "help_cmd_stack": "l Repite la selección [n] veces en la dirección seleccionada",
+  "copy_clipboard_not_empty": "ri Portapapeles de %s no está vacío, usar \"copy force\" para sobreescribir portapapeles actual",
```
I'd use `el portapapeles`

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "help_cmd_angel_new": "l Registra item actual como bloque ángel",
+  "action_angel": "bloque ángel",
+  "help_cmd_up": "l Teletransporta hacia arriba la cantidad de bloques especificada",
+  "help_cmd_brush_sphere": "l registra brocha para ahcer esferas de [radius] con [block]",
```
```suggestion
  "help_cmd_brush_sphere": "l registra brocha para hacer esferas de [radius] con [block]",
```

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "help_cmd_brush_ellipsoid": "l Registra brocha para crear elipsoide con radios [x_radius], [y_radius] y [z_radius] de [block]",
+  "help_pagination_page": "y Página %d ",
+  "help_cmd_brush_outline_force": "l Registra brocha para crear contornos, reeplazando cualquier bloque",
+  "help_cmd_brush_star": "l Registra brocha para crear prisma con base de estrella con [vertices] cantidad de lados",
```
I think `vertices` is not the number of sides. Not sure how to replace this though as I haven't really seen that brush yet.

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "angel_block_new": "w Cliquear con %s coloca un bloque ángel",
+  "action_hollow": "vaciado",
+  "help_cmd_walls_tooltip": "l Utiliza [sides] para elegir qué lados generar",
+  "help_cmd_drain": "l Drena el líquido en el que estás parado",
```
I would drop `parado`.

altrisi replied:
```suggestion
  "help_cmd_drain": "l Drena el líquido en el que estás",
```

altrisi replied:
```suggestion
  "help_cmd_drain": "l Drena el líquido en el que estás",
```

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "help_cmd_stack_tooltip": "g Si no se provee, la dirección aquella en la que el jugador está mirando",
+  "structure_delete_fail": "ri No se pudo eliminar %s.nbt, no existe el archivo",
+  "langs_availables": "y Idiomas disponibles:",
+  "help_your_selection": "c Tu slección",
```
```suggestion
  "help_your_selection": "c Tu selección",
```

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "help_cmd_redo_all": "l Rehace todos los movimientos deshechos",
+  "angel_block_given": "w Recibiste item de bloque ángel: %s",
+  "help_cmd_set_tooltip": "g Se peude utilziar una etiqueta para reemplazar bloques",
+  "help_cmd_outline_force": "l También reemplaza bloques que o son aire",
```
```suggestion
  "help_cmd_outline_force": "l También reemplaza bloques que no son aire",
```

altrisi reviewed (commented):
First of all, my region is Spain, so if something I said is regional and/or doesn't make sense, please ignore it directly.

Also some general things is I'd use a more personal language instead of infinitives (e.g. `usar`->`usa`/`use`) and I'd add some more `el`/`la`/`los` etc (don't know how to say `artículos` in English). (I ended up commenting only one or two, but it applies to more).

After seeing this and just the English language, I think I'll have to look at changing the way the help command works as to not make some messages so compressed.

Firigion commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "help_cmd_outline_tooltip": "g Especificar [outline_block] para sólo crear el contorno de formas de ese material",
+  "langs_not_found": "r Archivo de idioma %s no hallado",
+  "language_code": "es_la",
+  "help_cmd_stack_tooltip": "g Si no se provee, la dirección aquella en la que el jugador está mirando",
```
Damn, I was tired when I wrote this. I'll fix later. Thanks for reading all of these =D

Firigion commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "help_cmd_help": "l Muestra este menú de ayuda, o la página especificada",
+  "help_cmd_redo_all": "l Rehace todos los movimientos deshechos",
+  "angel_block_given": "w Recibiste item de bloque ángel: %s",
+  "help_cmd_set_tooltip": "g Se peude utilziar una etiqueta para reemplazar bloques",
```
I looked up how minecraft translates tag and I went with it. So etiqueta it is.

altrisi commented:
About infinitives, problem I see is that when giving instructions infinitive it sounds like you are talking about someone else e.g. "Sostiene un ítem" doesn't sound like it's telling me to hold the item, but that someone is holding it. I'd add them, probably yes, with variants if needed (I'll have to add language inheritance to make it be easier to update).

About articles: It may be just something regional, though it's not really unnatural, just I feel them less fluid.

Trying to be short on messages has to go though, it started on the help menu, that I'll have to change since some do really feel unnatural even in English, by (someday) finding a way to calculate how many lines does something occupy, but it is spreading everywhere.

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,171 @@
+  "brush_info_flags": "b Banderas: ",
+  "action_sphere": "esfera",
+  "help_cmd_paste": "l Pega portapapeles",
+  "help_cmd_brush_generic": "l Sostiene un ítem para covnertirlo en brocha",
```
```suggestion
  "help_cmd_brush_generic": "l Sostiene un ítem para convertirlo en brocha",
```

Firigion commented:
Implemented the suggested changes by adding a bunch of articles and conjugating some verbs. Also changed `vertices` argument type to `points` to make it more consistent when used in polygon and star prisms.

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,180 @@
+<	{
```
```suggestion
{
```

altrisi commented on se.data/langs/es_la.json:
```diff
@@ -0,0 +1,180 @@
+  "brush_info_flags": "b Banderas: ",
+  "brush_more_info_tooltip": "Cliquea para más información sobre la brocha %s",
+  "action_sphere": "esfera",
+<<<<<<< HEAD
+  "help_cmd_paste": "l Pega el portapapeles",
+  "help_cmd_brush_generic": "l Sostén un ítem para covnertirlo en brocha",
+  "angel_block_bad_item": "r El bloque ángel no puede ser algo que ya está asignado a otra acción",
+  "move_selection_no_player_error": "r Para mover la selección en la dirección del jugador hace falta un jugador",
+  "language": "Español (Latinoamérica)",
+=======
+  "help_cmd_paste": "l Pega portapapeles",
+  "help_cmd_brush_generic": "l Sostiene un ítem para convertirlo en brocha",
+  "angel_block_bad_item": "r Bloque ángel no puede ser algo que ya está asignado a otra acción",
+  "move_selection_no_player_error": "r Para mover la selección en la dirección del jugador, hace falta un jugador",
+  "language": "Español",
+>>>>>>> 5a700f9622b2bc3780e71d3c929db5844ab70ec0
```
Sounds like conflicts...

I guess the top one is the new one?

Firigion replied:
Yup, idk how I missed this.

altrisi reviewed (commented):
Seems like there were some remaining conflicts here.

Other than that, it looks good to me, I'd merge it already, things can be tweaked later if necessary.

Restored `en_us.json` since that is autogenerated anyway and else it may conflict with other PRs.

altrisi commented:
Please squash when merging. 25 commits that don't really add meaningful information on their own since they were generated by Github.
